packages/breadboard/package.json

```diff
@@ -159,6 +159,7 @@
     "node": ">=19.0.0"
   },
   "dependencies": {
+    "@google-labs/breadboard-schema": "*",
```

Comment:
Make this a real semver dependency; changesets will keep it sync'd automatically.

Reply:
```suggestion
    "@google-labs/breadboard-schema": "1.0.0",
```